app/validation/validator.py

```diff
@@ -26,7 +26,7 @@ def validate_schema(self, json_to_validate):  # noqa: C901  pylint: disable=too-
 
         errors = []
 
-        # errors.extend(self._validate_schema_contain_metadata(json_to_validate))
+        errors.extend(self._validate_schema_contain_metadata(json_to_validate))
```

Comment:
We're in an odd position now. Author _should_ be failing to preview questionnaires with piped values due to this. However, because author still uses dot notation instead of square brackets to pipe values it's still working.

I think we should allow both dot notation and square brackets - and we should validate both.

@samiwel FYI - any thoughts?

Reply:
@nealedj You can pipe using both dot or bracket notation. I've already made it to allow both.

Reply:
Sorry I was being daft and getting this mixed up with piped answers. Ignore me!